app/dashboard/router.py

```diff
@@ -194,11 +194,21 @@ def get_queryset(self):
             urls = self.request.query_params.get('github_url').split(',')
             queryset = queryset.filter(github_url__in=urls)
 
-        # filter by urls
+        # filter by orgs
         if 'org' in param_keys:
-            org = self.request.query_params.get('org')
-            url = f"https://github.com/{org}"
-            queryset = queryset.filter(github_url__icontains=url)
+            for orgs in param_keys:
```

Comment:
This will iterate through every parameter they send while making the request.
We don't seem to be doing anything with `orgs` here.  Can we eliminate that part to something like:

```python
# filter by orgs
val = self.request.query_params.get('org', '')
values = val.strip().split(',')
values = [value for value in values if value and val.strip()]
if values:
    _queryset = queryset.none()
    for value in values:
        org = value.strip()
        _queryset = _queryset | queryset.filter(github_url__icontains=f'https://github.com/{org}')
    queryset = _queryset
```

Reply:
perfect, I will make the changes here. Thanks @mbeacom